src/encode.ts

```diff
@@ -4,28 +4,29 @@ import { messageToTypeName, wrapperTypeName } from './types';
 import { LongOption } from './options';
 
 export function generateEncoder(ctx: Context, typeName: string): Code {
+  const { options } = ctx;
   const name = wrapperTypeName(typeName);
   if (!name) {
     return code`${messageToTypeName(ctx, typeName, { keepValueType: true })}.encode(value).finish()`;
   }
 
   if (name == 'Timestamp') {
-    const TimestampValue = imp(`${name}@./google/protobuf/timestamp`);
+    const TimestampValue = imp(`${name}@./google/protobuf/timestamp${options.fileSuffix}`);
```

Comment:
Wdyt about making an `imp` helper method in `src/utils` that looks something like:

```
imp(ctx, name, `./google/protobuf/timestamp`);
```

And then does the stitching together there? Might be nice to centralize the `fileSuffix` handling.

Reply:
@stephenh Ok, I created a helper `impProto` to consolidate the logic as described, the suffix avoids naming collisions since there are imports of public JS libraries too.  All of the imports had template literals with relative paths, so it's really a style call on whether you consider the `'./'` part of the module name or not.

There is also a circular dependency caused by makeUtils if the context is one of the function parameters. Just using the options avoids this.

Anyway, happy to make changes here if you have different preferences.